.github/workflows/build.yaml

```diff
@@ -34,6 +34,14 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  java-build:
```

Comment:
`build.yaml` is only used for building packages that we distribute as a part of nightly builds. Do you expect this to distribute a new conda package? It should probably be removed, see other comments below.

Reply:
I don't think we expect this to distribute a new conda package, so I have removed this bit